pkg/adapter/mtping/runner_test.go

```diff
@@ -206,6 +204,45 @@ func TestStartStopCron(t *testing.T) {
 
 }
 
+func TestStartStopCronDelayWait(t *testing.T) {
+	tn := time.Now()
+	seconds := tn.Second()
+	if seconds > 57 {
+		time.Sleep(time.Second * 4) // ward off edge cases
+	}
+	ctx, _ := rectesting.SetupFakeContext(t)
+	logger := logging.FromContext(ctx).Sugar()
```

Comment:
Are you certain this is not already a sugared logger? (sorry if I missed some important update in kn/pkg)

Reply:
func FromContext(ctx context.Context) *zap.Logger {
	return logging.FromContext(ctx).Desugar()
}